src/CoreShop/Component/Tracking/Tracker/CompositeTracker.php

```diff
@@ -31,7 +31,7 @@ public function isEnabled(): bool
             return $this->enabled;
         }
 
-        foreach ($this->trackerRegistry as $tracker) {
+        foreach ($this->trackerRegistry->all() as $tracker) {
```

Comment:
SA, the most boring colleague you want to have. :)

Reply:
;)